### ISSUE TITLE
Consider open source tools port (Kicad)

### ISSUE THREAD

**Issue by joulesverne** [wisp/wisp6-hardware#9]
For best developer/engineer/hobbyist accessibility, consider porting this design to Kicad, which should be able to manage a design of this complexity.
